### ISSUE TITLE
~

### ISSUE THREAD

**Issue by devnoname120** [Princess-of-Sleeping/SceKernelModulemgr-Reverse#2]
![is_this_uofw](https://user-images.githubusercontent.com/2824100/72193682-d66ce700-340a-11ea-9557-c9fe11091f3e.jpg)

**Comment by CelesteBlue-dev** [comment]
Yes and no. Yes because we will have our own PSVita REed OS. No because Vita3k seems not to need it prioritely. FAPS Team is too busy with PSVita native projects, but we will help Vita3k when we are done.
